rdap_types/src/lib.rs

```diff
@@ -123,28 +120,30 @@ impl<'de> Deserialize<'de> for CountryCode {
 }
 
 /// https://tools.ietf.org/html/rfc7483#section-4.2
-#[skip_serializing_none]
 #[derive(Serialize, Deserialize, Debug)]
 pub struct Link {
+    #[serde(skip_serializing_if = "Option::is_none")]
     pub value: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
     pub rel: Option<String>,
```

Comment:
These two are actually required per: https://www.rfc-editor.org/rfc/rfc9083.html#section-4.2-4

Complete changes from RFC7483
https://www.rfc-editor.org/rfc/rfc9083.html#name-changes-from-rfc-7483

Reply:
I left them optional for now to allow the 7483 interpretation, but added a comment about them being mandatory in 9083.